hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/storage/HoodieHFileConfig.java

```diff
@@ -43,9 +43,10 @@
   private final Configuration hadoopConf;
   private final BloomFilter bloomFilter;
   private final KeyValue.KVComparator hfileComparator;
+  private final String schemaKeyFieldId;
```

Comment:
Why is this an Id and not name? schemaKeyFieldName

Reply:
fixed this.